benchmark/utils/DriverUtils.swift

```diff
@@ -181,7 +176,7 @@ struct TestConfig {
 
       // We support specifying multiple tags by splitting on comma, i.e.:
       //
-      //  --tags=array,set
+      //  --tags=Array,Dictionary
```

Comment:
Instead of doing this, can you rename the categories to Array so that they are lower case. We should match swift style here.

Reply:
I'm not sure I follow. The categories seem logical to me at the moment: Capitals for classes, lowercase for descriptive categories. Here I have only updated the comment to correctly reflect the current categories.

Reply:
I would rather you just in a separate commit, just change the names of Array, Dictionary rather than update the comments.

Reply:
To be clear, the reason why I am against using caps here is that this is against the swift style guide. All enum cases should be camelCased. Why should the benchmark suite do something different from /all/ other swift projects. We should be standard here.

Reply:
I agree to @palimondo here: The categories which refer to concrete types in the stdlib should be named exactly as those types, everything else with lowercase.
As this is a user-visible name the swift enum case style is of minor interest.

Reply:
Ok. I am fine with dropping this. But just to be clear, I am not saying that the "user-visible" name shouldn't be capitalized. You would do it like this:

```
enum BenchmarkTag : String {
case array = "Array"
case string = "String"
case cpubench
}
```

I am just saying that IMO all code in the swift compiler should follow the swift style guide closely because our swift code is often times read as an example code by external developers. Whenever we have diverged in the past, our users have read the code and gotten confused. We are potentially inviting that confusion here.